### DISCUSSION TITLE
Syncthing issue, problem syncing from PC to hard drive connected to PI

### DISCUSSION THREAD

**Discussion by clothlust** [IceWhaleTech/CasaOS#618] [General]
**Describe the bug**
Unable to sync folder from PC to a Casa (installed on a PI). Error on syncthing: Failed to create folder root directory mkdir /mnt/homenas: permission denied, folder path is missing.

Its an external drive connected to the PI & jellyfin/navidrome works well; detecting the external drive & its content

**To Reproduce**

**Expected behavior**

**Screenshots**

![error](https://user-images.githubusercontent.com/96565791/153189155-39026553-3455-479f-a4fb-efe6589ab54b.jpg)

**Desktop (please complete the following information):**
 - OS: windows 11

**Additional context**

**Comment by HafdisE** [comment]
To add more information, it appears the `/DATA/` folder and `/mnt/` folders aren't mounted as volumes in the inbuilt syncthings container meaning the only place we can sync files from is the config folder somewhere in `AppData`.  When you look at the app store version the `/DATA/` folder is mounted as a volume by default. But not the inbuilt version.  This doesn't seem intentional.

**Comment by altShiftDev** [comment]
+1 as I'm also experiencing this issue. The in-built syncthings does not have access to `/root/DATA/mydrive` so it can never sync to an external drive.

A similar issue to OP happens when I try to install syncthings from the store. With the store installation you can access the container variables and add your own volume but Syncthings still has no write permissions to the mounted drive.

![image](https://user-images.githubusercontent.com/24980556/175437647-1a723124-62b2-4a9e-ae24-a2e45e269c07.png)

**Comment by tigerinus** [comment]
Converting this to discussion due to that Syncthing is no longer part of CasaOS feature set, due to its complexity to integrate. This issue should be considered as a question toward a CasaOS app, like many other CasaOS app questions.
